### ISSUE TITLE
Comments on the same line as a enum case don't get sorted alongside cases when enabling wrapEnumCases

### ISSUE THREAD

**Issue by nagra** [nicklockwood/SwiftFormat#1011]
Hi,

I've noticed when running the wrapEnumCases rule, comments on the same line as a enum case don't get sorted.
In some cases, comments are actually placed as new case statement themselves. 

The following example is output I received when running the rule on a enum which contained comments on both lines:

Before:
```
public extension TabSwitcherView {
    enum LayoutMode {
        case contentSizeWidth // layout is set to take horizontal space as needed by content
        case fullWidth // layout is set to stretch horizontally taking all available space
    }
}

```

After:
```
public extension TabSwitcherView {
    enum LayoutMode {
        case contentSizeWidth
        case // layout is set to take horizontal space as needed by content
            fullWidth // layout is set to stretch horizontally taking all available space
    }
}
```

**Comment by nicklockwood** [comment]
@nagra can you also include the code before it was formatted? I'm not completely clear how to recreate this issue.

**Comment by nagra** [comment]
@nicklockwood my bad, I've updated the original description 👍.

**Comment by nicklockwood** [comment]
@nagra I'm not able to recreate the bug with the example code you've provided. I'm also not sure why the `wrapEnumCases` rule would affect this code as it's already wrapped.

Do you have any other rules enabled? Or are you using any custom configuration?

**Comment by nicklockwood** [comment]
@nagra it's also possible something else in the same file might be causing the bug. Can you provide the whole file, or a cut-down file that still reproduces the issue?

**Comment by nicklockwood** [comment]
@nagra OK, I've managed to reproduce it with a different example:

```swift
    enum Foo {
            case bar, // bar
                 baz, // baz
                 quux // quux
        }
```

**Comment by nicklockwood** [comment]
@nagra fixed in 0.48.12
